src/Angular.js

```diff
@@ -893,7 +893,35 @@ var csp = function() {
   return (csp.isActive_ = active);
 };
 
+/**
+ * @ngdoc directive
+ * @name ng.directive:ngJq
+ *
+ * @element html
+ * @param {string} (optional) the name of the library available under `window` to be used for angular.element
```

Comment:
@mzgol Okay, this is starting to get weird.  If you look at my master right now, the changes are there: https://github.com/mboudreau/angular.js/blob/master/src/Angular.js